sklearn/metrics/_scorer.py

```diff
@@ -452,20 +469,15 @@ def _score(self, method_caller, clf, X, y, **kwargs):
             raise ValueError("{0} format is not supported".format(y_type))
 
         if is_regressor(clf):
-            y_pred = method_caller(clf, "predict", X)
+            response_method = self._response_method or "predict"
```

Comment:
nit: What do you think of making this a multi-liner like a few lines below for the classifier case? For me it would make the symmetry clearer/less easy to overlook that `or "predict"`.

Reply:
fair enough